### ISSUE TITLE
WP Demo Button no Longer working with 1.0.8

### ISSUE THREAD

**Issue by bearded-avenger** [WordPressDemo/WP-Demo#19]
The shortcode for the demo button doesn't seem to work anymore with 1.0.8.

is this correct?

```
[wp-demo parent_blog_id="4" submit_label="Submit"]Create a Demo[/wp-demo]
```

**Comment by bfintal** [comment]
Yup that's correct, tested it by placing that shortcode in a page of another site with the `parent_blog_id` belongs to another site. How is it not working in yours?

**Comment by bearded-avenger** [comment]
I'm using it on the home page of the site that's being cloned. Blog ID 1. I'm not using the parent_blog_id feature because that update broke the button.

It's not working for me locally or on the demo, its very easily replicable. had to revert back to 1.0.6.

I can't get the button to show in any way shape or form with or without attributes. The shortcode is parsing I can dump out the args but the button is nowhere to be found.

**Comment by bfintal** [comment]
Can you paste in here what you're using?

I have this in my local in the front page of my main site:

```
[wp-demo]The main demo[/wp-demo]
[wp-demo parent_blog_id='2']Try it out[/wp-demo]
```

Both are working fine, as well as the preview bar button. Connecting this with the other issue, both are also sending out activation emails.

I think the main problem here is that I can't replicate the issues on my end. I'm pretty sure we can fix the issues once I can replicate them. OR would you be able to set up some demonstration site outside your production one then set it up to make the problems show up, then give us access to the files? We can resort to that since I'm not seeing the errors for myself.

**Comment by bfintal** [comment]
Come to think of it, maybe there's a plugin or theme that's causing this. What plugins and themes are you using? In my local all I have Easy WP SMTP, Redux & WP Demo all network activated. I also tried with Redux deactivated. I'm using the default Twenty Fourteen theme during my tests.

**Comment by MMakijenko** [comment]
Same is happening to me. 1.0.6 shortcode works (outputs link), any higher version instead of the link outputs nothing.

**Comment by bfintal** [comment]
Hey Michael, have you tried with version 1.1? Yup we've had that problem
before but it should be working now

**Comment by MMakijenko** [comment]
Yes, I started with 1.0.4 worked all good. But then I updated to 1.1 and button disappeared. Then I downgraded versions one by one and on 1.0.6 shortcode started to work again. I'm using "plain" button shortcode without any params and I have problems both on local and live sites.
